snuba/clickhouse/formatter/expression.py

```diff
@@ -57,6 +57,12 @@ def __alias(self, formatted_exp: str, alias: Optional[str]) -> str:
             self.__parsing_context.add_alias(alias)
             return f"({formatted_exp} AS {escape_alias(alias)})"
 
+    def _format_string_literal(self, exp: Literal) -> str:
+        return self.__alias(escape_string(cast(str, exp.value)), exp.alias)
+
+    def _format_number_literal(self, exp: Literal) -> str:
+        return self.__alias(str(exp.value), exp.alias)
+
```

Comment:
I feel this could be cleaner if you had a base abstract class with these two abstract methods you cannot instantiate (call ClickhouseExpressionFormatterBse or something like that) and two concrete classes, one that provide the anonymized implementation and the other that provides the standard one.
This would implement a `template method` thus making it clear for subclasses what they must extend and what they cannot extend. So there is no chance somebody would add logic in this class in the extended method by mistake that would break the subclass.